components/utilities/layout.js

```diff
@@ -110,7 +110,7 @@
                 layoutNodes.style.height = height;
 
                 // use the Facebook CSS goodness
-                computeLayout(layoutNodes);
```

Comment:
I'd actually misread the diff and didn't spot that the method was now on the object... At least it's not a singleton factory adapter!

Let's leave it as it is, happy to merge.